HMCL/src/main/resources/assets/lang/I18N_zh.properties

```
@@ -409,6 +409,7 @@ folder.config=模組設定目錄
 folder.game=實例執行目錄
 folder.logs=日誌目錄
 folder.mod=模組目錄
+folder.datapack=資料包資料夾
```

在以前的 PR 中，我把繁中内所有相关的词汇都改成了「目錄」。
本來在简中内也想用「目录」的，但当时维护者提出了意见，所以简中目前全部使用「文件夹」。

```suggestion
folder.datapack=資料包目錄
```